README.md

```diff
@@ -1,3 +1,59 @@
 # Castle Crashers
 
-A CMPT-306 game made by a team of 6 people in the span of 1.5ish months!
\ No newline at end of file
+A CMPT-306 dungeon crawler game built in Unity and C# by a team of 7 people in the span of 1.5ish months!
+The name was a coincidence; we just like the ring to it (so did The Behemoth for the real Castle Crashers!)
+Note: the class focused on Game Mechanics, most of the visual and audio assets were used from free libraries or purchased from the Unity Store.
+
+## Overview
+The goal of the game is to traverse deeper and deeper to new levels of the castle, wield a variety of weapons to battle a variety of enemies, utilize gathered loot and coins to get stronger via opening chests, recieving permanent power upgrades, and buying stat & skill upgrades. Once the player reach the end of their journey, usually ended from a gruesome death, they would recieve a score based on their performance in the game.
+Link to our game trailer:
+https://youtu.be/27xrTJ3nenE
```

Comment:
We could/should maybe embed this directly since we can.

```html
<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/27xrTJ3nenE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Reply:
This i've never done before :o

Reply:
Yeah, just copy/paste that HTML in the comment into the Markdown and it should work (you might need an extra empty line on each side of it due to the Markdown processor needing to swap to HTML)

Reply:
I was reading that iframe isn't supported D: i did add a picture that links to the video tho

Reply:
Wow... GitHub way to be mean like that...

Reply:
But hey there is a link now at least! with a video :D